sale_fixed_discount/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 {
     "name": "Sale Fixed Discount",
     "summary": "Allows to apply fixed amount discounts in sales orders.",
-    "version": "12.0.1.0.0",
+    "version": "13.0.1.0.0",
     "category": "Sales",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Eficent, Odoo Community Association (OCA)",
```

Comment:
Could you take the chance to update the company name and any copyright in the module?

Reply:
Sure!